runner/src/rpc/tool.rs

```diff
@@ -96,3 +107,18 @@ pub(crate) fn credentials_filepath(account_id: AccountId) -> Result<PathBuf, Str
     path.push(format!("{}.json", account_id));
     Ok(path)
 }
+
+/// Convert `StateItem`s over to a Map<data_key, value_bytes> representation.
+/// Assumes key and value are base64 encoded, so this also decode them.
+pub(crate) fn into_state_map(state_items: Vec<StateItem>) -> HashMap<String, Vec<u8>> {
+    let decode = |s: StateItem| {
```

Comment:
why not have this closure return a result so that it doesn't panic on invalid response?